index.html

```diff
@@ -53,6 +54,241 @@ <h1>I'm Redwan.<br/>
+                            <ul>
+                                <li>
+                                    <a class="btn" href="">HTML</a>
+                                </li>
+                                <li>
+                                    <a class="btn" href="">CSS</a>
+                                </li>
+                                <li>
+                                    <a class="btn" href="">JavaScript</a>
+                                </li>
+                            </ul>
+                          </div>
+                          <div class="action">
+                            <button class="submit">See project</button>
+                        </div>
+                    </div>
+                </div>
+               </div>
+        </section>
+
+
+        <section class="section-bottom"> 
+            <div class="headerTop">
+                <h1>About Myself </h1>
+                <p >
+                    Hello I’m a software developer! I can 
+                    help you build a product , feature or 
+                    website. Look through some of my work 
+                    and experience! If you like what you see 
+                    and have a project you need coded, 
+                    don’t hesitate to contact me.
+                </p>
+            </div>
+            <!-- End of header top -->
+            <div class="headerBottom">
+                <a href="#">LET’S CONNECT</a>
+                <div class="link-btn">
+                    <ul>
+                        <li>
+                            <a href="#"><img src="images/link-Vector4.svg" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/Instagram2.png" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/link-Vector5.svg" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/Twitter.svg" alt="Twitter"/></a>
+                        </li>
+                        
+                        <li>
+                            <a href="#"><img src="images/link-Vector3.svg" alt=""/></a>
+                        </li>
+                        
+                       
+                    </ul>
+                </div>
+                <div class="action">
+                    <button class="submit">Get my resume</button>
+                </div>
+                <div class="tags-bottom">
+                    <div class="container-links">
+                        <ul>
+                            <li> Languages</li>
+                        </ul>
+                    </div>
+                    
+                    <div class="links">
+                        <div class="btn">
+                            <div class="link-log"><img src="Images/JS.png" alt="">  </div>
```

Comment:
- [ ] Kindly adhere to the design from the Figma, your design doesn't match it 👇🏽 👇🏽  Please be keen to all little details 🕵🏽 For example here your design is missing all the arrows 🔽 

![Screenshot from 2023-06-06 01-25-33](https://github.com/ethred/Portfolio/assets/10439283/4573a05b-1e8f-483c-8828-4f7b924ac5b3)

Design from Figma 👇🏽 👇🏽 

![Screenshot from 2023-06-06 01-25-16](https://github.com/ethred/Portfolio/assets/10439283/7fa4fe13-6817-4f06-997f-c0519b2efa62)